src/urls/urls-darklist.json

```diff
@@ -9482,5 +9482,49 @@
   {
     "id": "https://smartcontractlink.com",
     "comment": "SmartContract/LINK fake crowdsale site"
+  },
+  {
+    "id": "https://eosclassic.co",
+    "comment": "Phish: Fake EOS Classic token site"
+  },
+  {
+    "id": "https://tokenjar.io/EOSC",
+    "comment": "Phish: Fake EOS Classic token site"
+  },
+  {
+    "id": "https://etherdelta.com/#0xce3ad07c9c7a5f89d39eb1449913d3feecabcf8d-ETH",
```

Comment:
I'm not sure how others are using this list, with that in mind, Im not sure we can blacklist block explorers. I do recommend black listing the address on the address darklist as that shows on the current iteration of myetherwallet.

Reply:
@gamalielhere Will do, thanks for the review

Reply:
@eosclassicteam Same thing can be said with the social ones, (twitter, discord, etc)